roles/pulp_devel/tasks/install_podman.yml

```
@@ -45,20 +45,36 @@
   ignore_errors: '{{ ansible_facts.distribution_version == "8.1" }}'
 
 - name: Adding podman registries (Debian-specific)
-  command: echo -e "[registries.search]\nregistries = ['docker.io', 'quay.io']" | tee /etc/containers/registries.conf
+  # This should have been a toml_file module if I had found one
+  ini_file:
+    path: /etc/containers/registries.conf
+    section: registries.search
+    option: registries
+    value: "['docker.io', 'quay.io']"
   become: yes
-  args:
-    creates: /etc/containers
   when: ansible_facts.distribution == 'Debian'
 
+- name: Adding podman insecure registries (Debian-specific)
+  # This should have been a toml_file module if I had found one
+  ini_file:
+    path: /etc/containers/registries.conf
+    section: registries.insecure
+    option: registries
+    value: "['localhost:24817']"
+  become: yes
+  when:
+    - ansible_facts.distribution == 'Debian'
+    - pulp_webserver_disable_https | default(false)
+
 # Needed to allow non-root users to run podman containers on Debian
 - name: Enable unprivileged_userns_clone (Debian specific)
   sysctl:
     name: kernel.unprivileged_userns_clone
     value: "1"
+  become: yes
   when:
     - ansible_facts.distribution == 'Debian'
-    - ansible_facts.env.container | default("") == "podman"
+    - ansible_facts.virtualization_type | default("") not in ["docker", "podman"]
```

On Ansible 2.9 at least, this will not work. ansible_facts.virtualization_type does not understand podman. podman always sets the env vars container=podman and containers=podman though.

Let's just use this code from now on, it will be future-proof (in case ansible_facts.virtualization_type at some point understands podman), and it will also avoid whenever we set container=docker .

```suggestion
    - ansible_facts.virtualization_type | default("") not in ["docker", "podman"]
    - ansible_facts.env.container | default("") not in ["docker", "podman"]
```